libs/core/web/src/lib/routes/IsLoggedRoute.tsx

```diff
@@ -0,0 +1,13 @@
+import { useAuthContext } from '@form-exercise/utils';
+import { Navigate, Outlet } from 'react-router-dom';
+import { AppRoute, getRoutePath } from './routes';
+
+export const IsLoggedRoute = () => {
```

Comment:
the return value is not a boolean, it's a bit weird to prefix the name by `Is`, Name it LoggedRoute